### DISCUSSION TITLE
running quasar in ssr moded local/production with ssl

### DISCUSSION THREAD

**Discussion by felek000** [quasarframework/quasar#13848] [CLI - SSR mode]
Hello,
I use ssr mode and works well (vite). However i want create production mode and only think i found is:
https://quasar.dev/quasar-cli-webpack/developing-ssr/ssr-production-export#introduction

I generated certs using mkcert and add to ssl directory in src-pwa. On build directory is not  copy so i done it my self. 

Source of production export is:
```
import {ssrProductionExport} from 'quasar/wrappers'
import https from 'https'
import * as fs from 'fs'

const key = fs.readFileSync('./ssl/web.local-key.pem')
const cert = fs.readFileSync('./ssl/web.local.pem')

export default ssrProductionExport(({app, port, isReady}) => {
  return isReady().then(() => {
    const credentials = {key: key, cert: cert}
    const server = https.createServer(credentials, app)
    app.listen(port, () => {
      console.log('Server listening at port ' + port)
    })
  })
})
```

When i run no https page works but when i go same page https its show  page unavailable.
Is ssr mode is for production also ? And how can i add certs to it ?

**Comment by dmitry** [comment]
I suggest you use nginx or a similar proxy-server behind the node.
